### DISCUSSION TITLE
How to enable task progress (x of X) y% Completed like the showcase

### DISCUSSION THREAD

**Discussion by mnpqraven** [nvim-neorg/neorg#716] [Q&A] [answered]
https://user-images.githubusercontent.com/76052559/151614059-41b590cd-07ea-437c-84b9-536de6d1adfa.gif
How do I enable the option to show the task progress like the gif above (taken from the showcase in the README) ? I just started trying out Neorg recently and can't seem to find the option to turn this on. This is what I get after writing a basic list, would like a few pointers on how to set it up
![image](https://user-images.githubusercontent.com/8957788/213125948-54c49074-543f-4c77-a4d0-957dd1fbe66d.png)

I have the following modules in the neorg config:
```
require('neorg').setup({
  load = {
    ["core.defaults"] = {}, -- Loads default behaviour
    ["core.norg.concealer"] = {
      config = { icon_preset = "diamond" }
    },
    ["core.norg.dirman"] = {
      config = {
        workspaces = { test = "~/dev_env/rust-local/pnc-checklist" },
    },
    ["core.norg.completion"] = {
      config = { engine = "nvim-cmp" }
    },
    ["core.keybinds"] = {
      config = {
        default_keybinds = true,
        neorg_leader = "<leader>"
      }
    },
  },
})
```

**Comment by max397574** [comment, marked as answer]
this was temporarily removed
it's being worked on here
<https://github.com/nvim-neorg/neorg/tree/todo-introspector>

**Reply by mnpqraven** [reply]
I see, thank you very much !
